docs/guides/admin/docs/configuration/basic.md

```diff
@@ -64,7 +64,16 @@ broker is mostly run on the admin server of Opencast but can be run separately.
 For more details about the setup, have a look at the [Apache ActiveMQ configuration guide](message-broker.md).
 
 
-Step 5: Database Configuration
+Step 5: Setting up Elasticsearch
+--------------------------------
+
+Since version 9.0, Opencast requires running an external Elasticsearch instance for indexing purposes.  Typically this
+runs on the admin node, although that is not required.  Instructions for installing Elasticsearch can be found in the
+[installation documentation](../installation/index.md).  Fore more details about the setup, have a look at the
+[Elasticsearch configuration guide](elasticsearch.md).
```

Comment:
This guide is about basic configuration but this step is no configuration.
Do we even need any configuration if we recommend running Elasticsearch on the admin node?

Reply:
This step is the same as the ActiveMQ step immediately before it.  We don't need to make any config changes, as long as the admin node hosts the ES instance.  What about rewriting the last sentence to something like

`If you run your Elasticsearch instance on your all-in-one, or admin node then you do not need to make any configuration changes.  If you are *not* running Elasticsearch on that node, then please refer [this configuration guide](elasticsearch.md).`